spec/08-pattern-matching.md

```diff
@@ -753,12 +753,12 @@ already a variable or wildcard pattern.
 
 ###### Example
 Here is a method which uses a fold-left operation
-`/:` to compute the scalar product of
+`foldLeft` to compute the scalar product of
```

Comment:
Sufficient text is
```
uses `foldLeft` to compute
```

Reply:
This is a continuous and complete sentences

```
Here is a method which uses a fold-left operation `foldLeft` to compute the scalar product of two vectors:
```

Line breaks in Markdown file are just to make it easier to view text while editing.

Reply:
OK, IMHO "a fold-left operation foldLeft" doesn't sound good, but I don't know the term from classical rhetoric to describe this species of redundancy.

Reply:
You're right, then how about this one ?

Reply:
"Here uses a method" isn't grammatically correct.  Perhaps something along the lines of:

```scala
Here's an example which uses
`foldLeft` to compute the scalar product of
two vectors:
```

Reply:
Thanks for your correction, I will rebase soon.